### PR TITLE
Closes #23: add audio streaming to telegram with watson t2s

### LINKED ISSUE #23: Integrate WT2S
As a user, I want to be able to receive spoken audio messages from the bot, because this is a project requirement

### PR DESCRIPTION
**Pull request by xdivby0**

### CODE REVIEW

File: app/services/watsonSpeech.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const TextToSpeechV1 = require("ibm-watson/text-to-speech/v1");
+const {IamAuthenticator} = require("ibm-watson/auth");
```
Comment (oliver-rudzinski) [inline app/services/watsonSpeech.js]: `{ IamAuthenticator }` would be better.

File: app/services/watsonSpeech.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+const fs = require("fs");
+
+module.exports = function() {
```
Comment (oliver-rudzinski) [inline app/services/watsonSpeech.js]: Use ES6 callback function declaration: `() => {`

File: app/services/watsonSpeech.js (hunk 1 above)
Comment (xdivby0) [reply app/services/watsonSpeech.js]: source?

File: app/services/watsonSpeech.js (hunk 2 above)
Comment (xdivby0) [reply app/services/watsonSpeech.js]: We use `.eslintrc.json` for rules concerning formatting. Please make according changes to this file if you want to enforce specific code styles, refactor the existing files to match this style and create a new branch.

**Comment by xdivby0** [comment]
> 
> 
> Change all `()=>{ ... }` to `() => { ... }` (spaces before and after the arrow)

We use `.eslintrc.json` for rules concerning formatting. Please make according changes to this file if you want to enforce specific code styles, refactor the existing files to match this style and create a new branch.

File: app/services/watsonSpeech.js (hunk 1 above)
Comment (oliver-rudzinski) [reply app/services/watsonSpeech.js]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

Comment (xdivby0) [reply app/services/watsonSpeech.js]: `import {myExport} from '/modules/my-module.js';`
Source: your link

**Pull request merged**